modules/installation-guide/examples/system-variables.adoc

```diff
@@ -95,57 +95,55 @@ pass:[<!-- vale off -->]
 ,=== 
  Environment Variable Name,Default value, Description 
  
- `+CHE_INFRA_KUBERNETES_MASTER__URL+`,"","Configuration of Kubernetes client that Infra will use" 
- `+CHE_INFRA_KUBERNETES_TRUST__CERTS+`,"","Configurationof {orch-name} client that Infra will use" 
- `+CHE_INFRA_KUBERNETES_SERVER__STRATEGY+`,"`+multi-host+`","Defines the way how servers are exposed to the world in {orch-name} infra. List of strategies implemented in {prod-short}: default-host, multi-host, single-host" 
- `+CHE_INFRA_KUBERNETES_SINGLEHOST_WORKSPACE_EXPOSURE+`,"`+native+`","Defines the way in which the workspace plugins and editors are exposed in the single-host mode. Supported exposures: - 'native': Exposes servers using Kubernetes Ingresses. Works only on {kubernetes}. - 'gateway': Exposes servers using reverse-proxy gateway." 
- `+CHE_INFRA_KUBERNETES_SINGLEHOST_WORKSPACE_DEVFILE__ENDPOINT__EXPOSURE+`,"`+multi-host+`","Defines the way how to expose devfile endpoints, thus end-user's applications, in single-host server strategy. They can either follow the single-host strategy and be exposed on subpaths, or they can be exposed on subdomains. - 'multi-host': expose on subdomains - 'single-host': expose on subpaths" 
+ `+CHE_INFRA_KUBERNETES_MASTER__URL+`,"","Configuration of Kubernetes client master URL that Infra will use." 
+ `+CHE_INFRA_KUBERNETES_TRUST__CERTS+`,"`+false+`","Boolean to configure Kubernetes client to use trusted certificates." 
+ `+CHE_INFRA_KUBERNETES_SERVER__STRATEGY+`,"`+multi-host+`","Defines the way how servers are exposed to the world in Kubernetes infra. List of strategies implemented in {prod-short}: `default-host`, `multi-host`, `single-host`." 
+ `+CHE_INFRA_KUBERNETES_SINGLEHOST_WORKSPACE_EXPOSURE+`,"`+native+`","Defines the way in which the workspace plugins and editors are exposed in the single-host mode. Supported exposures: `native`:: Exposes servers using Kubernetes Ingresses. Works only on Kubernetes. `gateway`:: Exposes servers using reverse-proxy gateway." 
```

Comment:
aren't we extracting "Kubernetes" as a {something-variable} in all occurrences now?